src/server/index.ts

```diff
@@ -877,7 +892,12 @@ export class DocumentDriveServer extends BaseDocumentDriveServer {
             clipboard: [],
             state: state ?? document.state
         };
-        await this.storage.createDocument(driveId, input.id, documentStorage);
+        await this.storage.createDocument(
+            driveId,
+            input.id,
+            documentStorage,
+            input.synchronizationUnits
```

Comment:
We were already receiving the synchronization units when creating a document, but we were not doing anything with them.